supportedBackends/sfs/src/test/scala/cromwell/backend/sfs/SharedFileSystemJobExecutionActorSpec.scala

```diff
@@ -48,13 +48,24 @@ class SharedFileSystemJobExecutionActorSpec extends TestKitSuite
     val expectedOutputs: CallOutputs = WomMocks.mockOutputExpectations(Map("hello.salutation" -> WomString("Hello you !")))
 
     val expectedResponse = JobSucceededResponse(mockBackendJobDescriptorKey, Some(0), expectedOutputs, None, Seq.empty, None, resultGenerationMode = RunOnBackend)
-    val runtime = if (docker) """runtime { docker: "ubuntu:latest" }""" else ""
+    val runtime = if (docker) s"""runtime { docker: "$dockerImageUbuntu" }""" else ""
     val workflowDescriptor = buildWdlWorkflowDescriptor(HelloWorld, runtime = runtime)
     val workflow = TestWorkflow(workflowDescriptor, TestConfig.backendRuntimeConfigDescriptor, expectedResponse)
     val backend = createBackend(jobDescriptorFromSingleCallWorkflow(workflow.workflowDescriptor, Map.empty, WorkflowOptions.empty, runtimeAttributeDefinitions), workflow.config)
     testWorkflow(workflow, backend)
   }
 
+  private val dockerImageUbuntu = "ubuntu:latest"
```

Comment:
TOL: do we want to pin this rather than let it float?

Reply:
ToL: Future ticket? That's what the current test has been doing for months (years?).

What changed was: https://www.docker.com/increase-rate-limits

Reply:
Edit: To be clear- when docker introduced the rate limits, we stopped just `docker pull ubuntu` in ALL 30 of the CI environments. Ex: Papi centaur tests never needed ubuntu to be locally available, and the test were failing because the extra `docker pull ubuntu` calls were counting towards the limits in the link above.

But when that was removed, these tests started to fail because `docker run ubuntu ...` was pulling the image during the test's 20 second timeout.

Whether or not the tag should be pinned to a version is a good question (does the test always want to trying out the most recent `ubuntu`??), but IMHO a separate refined issue.